### ISSUE TITLE
man iocage shows update takes snapshot, snapshot not taken

### ISSUE THREAD

**Issue by agshekeloh** [iocage/iocage#618]
```sh-session
$ iocage --version
Version 1.0 ALPHA 1
$ uname -a
FreeBSD storm 12.0-ALPHA6 FreeBSD 12.0-ALPHA6  r338724  amd64
```

Man iocage shows:

     update        Runs freebsd-update to update the specified jail to the
                   latest patch level.  A backup snapshot is automatically
                   created to provide a rollback option.

I already have one snapshot. After the update, I would expect a second snap.

```
# iocage snaplist www1
+--------------+-----------------------+-------+------+
|     NAME     |        CREATED        | RSIZE | USED |
+==============+=======================+=======+======+
| install      | Thu Sep 20 17:34 2018 | 92K   | 56K  |
+--------------+-----------------------+-------+------+
| install/root | Thu Sep 20 17:34 2018 | 680M  | 192K |
+--------------+-----------------------+-------+------+
# iocage update www1

* Updating www1 to the latest patch level...
Looking up update.FreeBSD.org mirrors... 2 mirrors found.
Fetching public key from update4.freebsd.org... done.
Fetching metadata signature for 9.3-RELEASE from update4.freebsd.org... done.
Fetching metadata index... done.
Fetching 2 metadata files... done.
Inspecting system... done.
Preparing to download files... done.
Fetching 2545 patches.....

...nap...


/var/db/mergemaster.mtree

WARNING: FreeBSD 9.3-RELEASE HAS PASSED ITS END-OF-LIFE DATE.
Any security issues discovered after Sat Dec 31 19:00:00 EST 2016
will not have been corrected.
Installing updates... done.
ioc-www1: removed

www1 has been updated successfully.
# iocage snap^Gl^Gi^Gst www1
+--------------+-----------------------+-------+------+
|     NAME     |        CREATED        | RSIZE | USED |
+==============+=======================+=======+======+
| install      | Thu Sep 20 17:34 2018 | 92K   | 60K  |
+--------------+-----------------------+-------+------+
| install/root | Thu Sep 20 17:34 2018 | 680M  | 208K |
+--------------+-----------------------+-------+------+
```

I would expect the rollback snapshot to show up here? No snap in zfs list -t snap either

Not sure if doc, code, or user bug.

**Comment by skarekrow** [comment]
@Mrt134 update does not take snapshots, upgrade does IIRC

**Comment by rm76** [comment]
Same here with iocage Version 1.0 ALPHA 1 on FreeNAS 11.2.
No snapshot was taken at iocage update.
So no snapshot to rollback from either.
Was this fixed in newly-released version 1.1?
Thanks.
